changelogs/fragments/77-filter-droplets-by-tag.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - digitalocean - filter droplets by tag in dynamic inventory plugin
```

Comment:
Mind rewriting this as `- digitalocean - filter droplets by tag in dynamic inventory plugin (https://github.com/ansible-collections/community.digitalocean/pull/77)` please?

Reply:
OK